packages/ERTP/src/amountMath.js

```diff
@@ -1,13 +1,12 @@
+import { quote as q, throwRedacted as Fail } from '@endo/errors';
```

Comment:
This pattern is what I was trying to get away from with making a package at all. There's no export of `Fail` so there's no autocomplete when someone wants to import `Fail`.

`q` and `Fail` are the convention in this repo, so they should have exports. I propose `@agoric/errors` assert continue to export those.

The other changes in the PR look good. I'd have accepted without the errors changed. That's one of the good reasons to separate commits. E.g. `refactor: modernize @agoric/store imports` and separately `refactor: modernize errors imports`. Then we could easily pick which changes to include.

Incidentally, how did 6e0290e990066bf86a18bc1798ac927457530c1e turn into a49db1068bc575e60fbbee4394eebe5302bd5973 ?  Why not simply cherry-pick it?

Reply:
Incidentally, the plan of record is for `@agoric/assert` to itself import `@endo/errors`,
- https://github.com/Agoric/agoric-sdk/issues/5672